config/settings.json

```diff
@@ -1,11 +1,11 @@
 {
   "schema": { 
-      "timelines": "timelines",
-      "impressions": "impressions",
-      "htmls": "htmls",
-      "suggestions": "suggestions",
-      "accesses": "accesses",
-      "supporters": "supporters"
+      "timelines": "timelines2",
```

Comment:
`schema` maps the *logical* name of the collections to their *physical* name in the database?
If so, we might consider to version the documents, not the collections, by adding a `version` key to every document.